components/title/basic/src/index.js

```diff
@@ -0,0 +1,52 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+export default function TitleBasic ({align, className, subtitle, title}) {
+  function buildAlignClassName (alignProp) {
+    let alignClassName = ''
+    if (alignProp) {
+      alignClassName = 'sui-TitleBasic--align' + alignProp.charAt(0).toUpperCase() + alignProp.slice(1)
```

Comment:
This is just a method to build the modifier classname to apply to the component (according to SUIT naming convention, it must be separated with "--" prefix and written in camel case).
I can reduce this method, but I don't think it will help since obfuscate is not the same as simplify.
I will add a comment on this function to specify its main purpose.